api_parser.py

```diff
@@ -0,0 +1,34 @@
+import requests
+from datetime import datetime, timedelta
+
+
+def MakeApiRequest(start_date):
+    '''
+    This function works as follows:
+    1. Generate daily list of dates from start_date
+    2. For every day it makes api request to get USD to EUR exchange rate
+    3. After receiving a response it saves it to file
+
+    :param start_date: start date
+    :type start_date: int
+
+    :return None:
+    '''
+    # Generate daily date from start date
+    print('Start parsing ...')
+    start = datetime.strptime(start_date, '%Y-%m-%d')
+    end = datetime.now()
+    days = (end - start).days
+    for diff in range(days):
+        # Make API request for every date
+        parsing_date = start + timedelta(diff)
+        print("Parsing: https://api.exchangeratesapi.io/" + str(parsing_date.date()))
+        r1 = requests.get("https://api.exchangeratesapi.io/" + str(parsing_date.date()), {'base': "USD"})
```

Comment:
another thing that I lost to mention, we can create methods for each component like
read and write methods instead of having it under a single method

For better approach, creating a API class and having the methods inside. I believe I possibly haven't thought too much during the call.